Cargo.toml

```
@@ -53,6 +53,7 @@ wasmparser = "0.226"
 reqwest = { version = "0", default-features = false, features = [
   "rustls-tls-native-roots",
 ] }
+indicatif = "0.17.11"
```

We can be more relaxed about the patch version

```suggestion
indicatif = "0.17"
```

---

fixed